test/browser/run-test.sh

```diff
@@ -56,6 +56,11 @@ if [ "${TEST_OS#centos-8}" != "$TEST_OS" ]; then
     TEST_ALLOW_JOURNAL_MESSAGES="${TEST_ALLOW_JOURNAL_MESSAGES},couldn't create runtime dir: /run/user/1001: Permission denied"
 fi
 
+# HACK: https://github.com/systemd/systemd/issues/24150
+if [ "$ID" = "fedora" ]; then
+       TEST_ALLOW_JOURNAL_MESSAGES="Journal file /var/log/journal/*/user-1000@*.journal corrupted, ignoring file .*"
```

Comment:
You need to append, like the other cases do. Please add the missing `${TEST_ALLOW_JOURNAL_MESSAGES},`

Reply:
Oh I am so silly, yes!